noxfile.py

```diff
@@ -18,4 +18,4 @@ def lint(session):
     session.run("flake8", *args)
 
 
-nox.options.sessions = ["lint"]
+nox.options.sessions = ["black", "lint"]
```

Comment:
Whilst this does run black, the job won't be able to push the changes it makes as main is protected

Reply:
well that's problematic. It does address the issue locally.